components/SiteCard.tsx

```diff
@@ -15,83 +14,68 @@ interface SiteCardProps {
-            <p className="text-ocf-gray-500 text-xs font-medium">
-              {`Current output: ${
-                currentOutput != undefined
-                  ? currentOutput.toFixed(2) + ' kW'
-                  : 'loading...'
-              }`}
-            </p>
-            {installed_capacity_kw && (
-              <p className="text-ocf-gray-500 font-medium text-xs">
-                Max. capacity: {installed_capacity_kw.toFixed(2)} kW
+      <div className="relative overflow-hidden rounded-lg w-full">
+        <a
+          onClick={onClick}
+          ref={ref}
+          href={!isEditMode ? href : '#0'}
+          className={`h-fit w-full max-w-lg flex items-center bg-ocf-black-500 font-bold overflow-hidden ${
+            isEditMode && 'pointer-events-none'
+          }`}
+        >
+          <div className="flex flex-col flex-1 w-[60%] p-4 pl-5">
+            <h2 className="text-amber text-xl font-semibold">
+              {isLoading ? 'Loading...' : client_site_name ?? 'My Site'}
+            </h2>
+            <div className="flex flex-col mt-2 gap-1">
+              <p className="text-ocf-gray-500 text-xs font-medium">
+                {`Current output: ${
+                  currentOutput != undefined
+                    ? currentOutput.toFixed(2) + ' kW'
+                    : 'loading...'
+                }`}
               </p>
-            )}
-            <p className="text-ocf-gray-500 font-medium text-xs">
-              {`Current yield:
+              {installed_capacity_kw && (
+                <p className="text-ocf-gray-500 font-medium text-xs">
+                  Max. capacity: {installed_capacity_kw.toFixed(2)} kW
+                </p>
+              )}
+              <p className="text-ocf-gray-500 font-medium text-xs">
+                {`Current yield:
             ${
               installed_capacity_kw && currentOutput != undefined
                 ? (currentOutput / installed_capacity_kw).toFixed(2) + '%'
                 : 'loading...'
             }`}
-            </p>
+              </p>
+            </div>
           </div>
-        </div>
-
-        <div
-          className={`justify-center self-center mr-5 transition-all overflow-hidden max-w-[250px] ${
-            !isEditMode ? 'flex-1' : 'flex-0 w-0'
-          } duration-[${transitionDuration}ms]`}
-          ref={animationElement}
-        >
-          {displayGraph && <SiteGraph siteUUID={siteUUID} />}
-        </div>
 
+          <div className={`mr-5 w-[40%] pointer-events-none`}>
+            {/* TODO: find out why this left is necessary */}
+            <div className="relative -left-7">
+              <SiteGraph siteUUID={siteUUID} hidden={isEditMode} />
+            </div>
+          </div>
+        </a>
         <div
-          className={`transition-all ${
-            isEditMode ? 'w-4/12' : 'w-0'
-          } duration-[${transitionDuration}ms] flex translate-x-40`}
+          // @ts-ignore
+          inert={!isEditMode ? '' : null}
```

Comment:
think the `@ts-ignore` is fine for now—tried to fix it but couldn't get anything. however, isn't `inert` a boolean attribute? 

```suggestion
          // @ts-ignore
          inert={!isEditMode ? false : true}
```
Don't know for sure if that'd be correct behavior.

Reply:
I think tried just dropping the boolean in itself and ran into problems. This github comment seemed to simply this was the best way, though I could do '' : false. https://github.com/WICG/inert/issues/58#issuecomment-618016847